doc/book/connectors/__python.rst

```diff
@@ -39,8 +39,221 @@ For an example of using Python API with
+
+    *   -   Schema reload, Access tuple fields by names
+        -   Yes
+        -   FIXME
+        -   FIXME
+        -   FIXME
+
+    *   -   `SQL support <https://www.tarantool.io/en/doc/latest/reference/reference_sql/>`__
+        -   Yes (tests/test_op_sql.py)
+        -   No (using call server-side functions)
+        -   No
+        -   Yes (tarantool/connection.py)
+
+    *   -   `Interactive transactions <https://www.tarantool.io/en/doc/latest/book/box/stream/>`__
+        -   No
+        -   No
+        -   No
+        -   No (`issue #163 <https://github.com/tarantool/tarantool-python/issues/163>`__)
+
+    *   -   `Varbinary support <https://www.tarantool.io/en/doc/latest/book/box/data_model/>`__
+        -   No
+        -   No
+        -   No
+        -   No
+
+    *   -   `UUID support <https://www.tarantool.io/en/doc/latest/book/box/data_model/>`__
+        -   No
+        -   No
+        -   No
+        -   No
+
+    *   -   `Decimal support <https://www.tarantool.io/en/doc/latest/book/box/data_model/>`__
+        -   Almost (v2 branch)
+        -   No
+        -   No
+        -   No
+
+    *   -   `EXT_ERROR support <https://www.tarantool.io/ru/doc/latest/dev_guide/internals/msgpack_extensions/#the-error-type>`__
+        -   No
+        -   No
+        -   No
+        -   No
+
+    *   -   `[Soon] <https://github.com/tarantool/tarantool/discussions/6244>`__ Datetime support
+        -   No
+        -   No
+        -   No
+        -   No
+
+    *   -   `box.session.push() responses <https://www.tarantool.io/ru/doc/latest/reference/reference_lua/box_session/push/>`__
+        -   Yes (see push_subscribe option and docs/pushes.rst)
+        -   No
+        -   No
+        -   No
+
+    *   -   `Session settings <https://www.tarantool.io/en/doc/latest/reference/reference_lua/box_space/_session_settings/>`__
+        -   No
+        -   No
+        -   No
+        -   No
+
+    *   -   `Graceful shutdown <https://github.com/tarantool/tarantool/issues/5924>`__
+        -   No
+        -   No
+        -   No
+        -   No
+
+    *   -   `IPROTO_ID (feature discovering) <https://github.com/tarantool/doc/issues/2419>`__
+        -   No
+        -   No
+        -   No
+        -   No
+
+    *   -   Support `CRUD <https://github.com/tarantool/crud>`__
+        -   No
+        -   No
+        -   No
+        -   No
+
+    *   -   Support retrying when appropriate
+        -   No
+        -   No
+        -   No
+        -   No
+
+    *   -   Implicit reconnecting strategy
+        -   autoreconnect
+        -   No
+        -   Yes? (reconnect_max_attempts, reconnect_delay)
+        -   Yes? (reconnect_max_attempts, reconnect_delay), checking of connection liveness
+
+    *   -   Round robin failover
+        -   No
+        -   No
+        -   No
+        -   Yes (see tarantool/mesh_connection.py)
+
+    *   -   `[Soon] <https://github.com/tarantool/tarantool/issues/6256>`__ Master discovering
```

Comment:
I would not link this issue here, see https://github.com/tarantool/doc/pull/2484#discussion_r772349714.

Reply:
Could you clarify what do you mean by that link (GH failed to show a comment body in PR if comment was resolved)?

Reply:
![image](https://user-images.githubusercontent.com/92749/147664676-0afa290f-dc09-4062-807a-e2ef8f492a24.png)

Reply:
The short explanation for those who will read it: there is a 'master discovery' used in connectors, and there is another 'master discovery' that PM's planned to implement in Tarantool. To avoid confusion, we don't need to mix up both 'master discovery'. It is a reason why link should be removed.